Project-Hotpot/Project-Hotpot/APIManager.swift

```diff
@@ -73,7 +73,7 @@ class APIManager: NSObject {
         // Set the playURI to a non-nil value so that Spotify plays music after authenticating
         // otherwise another app switch will be required
         // for now, leaving this line commented out purposefully
-        // configuration.playURI = ""
+         configuration.playURI = "spotify:track:7p5bQJB4XsZJEEn6Tb7EaL"
```

Comment:
Notice how there is a comment to leave this line commented out purposefully, then a commented out line, and then a filled line all back-to-back.
Keeping your code clean and concise will help reduce potential confusion later.